<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GroupCollectorTab</class>
 <widget class="QWidget" name="GroupCollectorTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>群组搜集器</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>账号列表</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="accountTableWidget"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="selectAllAccountsButton">
       <property name="text">
        <string>全选账号</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deselectAllAccountsButton">
       <property name="text">
        <string>反选账号</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="keywordInput"/>
     </item>
     <item>
      <widget class="QPushButton" name="addKeywordButton">
       <property name="text">
        <string>添加关键词</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="keywordList"/>
   </item>
   <item>
    <widget class="QPushButton" name="removeKeywordButton">
     <property name="text">
      <string>删除选中关键词</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>群组信息</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="groupTableWidget"/>
   </item>
   <item>
    <widget class="QPushButton" name="startCollectionButton">
     <property name="text">
      <string>开始采集</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="logTextEdit"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>